src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![allow(clippy::type_complexity)]
+#![allow(unstable_name_collisions)]
```

Comment:
where this is used?

Reply:
`.inspect_err` is in stdlib but not stabilized (no idea why), that's why there is a `result_inspect` crate exist, but you have to add this `allow` to keep cargo from warnings